src/client/audiomanager.js

```diff
@@ -2,7 +2,10 @@ import WZManager from './wzmanager';
 
 const AudioManager = {};
 
-AudioManager.playBGM = async function(name) {
+var musicVolume = 1;
```

Comment:
It's not recommended to use var. Try using const instead

Reply:
This is correct. Take a look at this style guide: https://github.com/airbnb/javascript.

Reply:
As an addendum, I don't recommend storing these variables at the module level. I prefer assigning these variables to the `AudoManager` object.